common/web/types/src/kmx/kmx-plus/kmx-plus.ts

```diff
@@ -409,6 +409,8 @@ export class TranTransform {
   to: StrsItem;
   mapFrom: StrsItem; // var name
   mapTo: StrsItem; // var name
+  _from?: string; // for serialization
+  _to?: string; // for serialization
```

Comment:
Will these cause any maintenance confusion with `from` and `to`?

Reply:
I'm using the same pattern as in kmx-plus-builder where we have

```js
  id: BUILDER_STR_REF; // str with original key id
  _id: string; // original key id, for sorting
```

https://github.com/keymanapp/keyman/blob/feat/developer/12874-kmx-to-xml-epic-ldml-editor/developer/src/common/web/utils/src/types/kmx/kmx-plus-builder/build-keys.ts#L23

I will add clearer documentation on the property.